assets/src/edit-story/components/inspector/document/status.js

```diff
@@ -69,17 +67,20 @@ function StatusPanel() {
       name: __('Draft', 'web-stories'),
       helper: __('Visible to just me', 'web-stories'),
     },
-    {
+  ];
```

Comment:
If `Draft` is the only option available in radios then better to not show the radio at all. Might be a bit odd seeing a radio selection with just one option. WDYT?

Reply:
We don't really have UX for this now. The UX might not be the best, but good enough for now.

Reply:
Sounds good to me to keep as-is for now but please follow up with @samitron7 so that she would be aware of this.